pkg/drivers/kic/oci/oci.go

```diff
@@ -524,16 +524,15 @@ func PointToHostDockerDaemon() error {
 	p := os.Getenv(constants.MinikubeActiveDockerdEnv)
 	if p != "" {
 		klog.Infof("shell is pointing to dockerd inside minikube. will unset to use host")
-	}
-
-	for i := range constants.DockerDaemonEnvs {
-		e := constants.DockerDaemonEnvs[i]
-		err := os.Setenv(e, "")
-		if err != nil {
-			return errors.Wrapf(err, "resetting %s env", e)
+		for i := range constants.DockerDaemonEnvs {
+			e := constants.DockerDaemonEnvs[i]
+			err := os.Setenv(e, "")
```

Comment:
There's an os.UnsetEnv command you can use here.

Reply:
Thanks, I just placed this for loop inside the if block and didn't look to improve the logic itself. Using Unsetenv is better, so I modified it.